.github/workflows/build.yml

```diff
@@ -14,11 +12,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Build
-      run: make ci
-    - name: Deploy
-      if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
-      uses: peaceiris/actions-gh-pages@v3
+    - uses: w3c/spec-prod@v2
       with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        publish_dir: ./out
+        TOOLCHAIN: bikeshed
+        SOURCE: spec.bs
+        GH_PAGES_BRANCH: gh-pages
+        BUILD_FAIL_ON: warning
```

Comment:
@dlaliberte wanted to turn off failing the build on warnings.  I'm generally a fan of requiring no warnings on builds.  Daniel, do you have a strong preference here?  maybe the w3c/spec-prod will give more useful warnings.

Reply:
I only wanted to turn off warnings because I couldn't figure out how to deal with all the warnings in #351.  But I have now dealt with them, so perhaps it is better if we can continue to avoid warnings.   I can build locally now too, so I can see all the warnings rather than just one at a time.